<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_record</class>
 <widget class="QWidget" name="Edit_record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="year">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QLabel" name="discription_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>375</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>描述:</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时间:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="day">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>259</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="category">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类别:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="type">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amount_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>金额:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编辑记录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="discription">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="in_or_out">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>支出/Expense</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>收入/Income</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Transaction_type">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>收入类型:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="index">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="index_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>记录编号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_report">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="year_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="month_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
